26-register_publication/1656-creator_navigates_to_the_reference_tab_and_selects_reference_subtype_journal_article.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Creator navigates to the Reference tab and selects Reference subtype "Journal article"
+
+    @1625
+    Scenario: Creator navigates to the Reference tab and selects Reference subtype "Journal article"
+        Given that a Creator navigates to the Reference tab
+        And they have selected "Contribution to journal" as Reference type
+        When they select "Journal article" as Reference Subtype
+        Then they see mandatory fields:
+            | Search-box for Journal |
+            | Peer reviewed          |
+        And they see optional fields:
+            | DOI            |
+            | Volume         |
+            | Issue          |
+            | Pages from     |
+            | Pages to       |
+            | Article number |
```

Comment:
samme kommentar som jeg gjorde på forrige PR. Kan vi skrive disse featurene på samme format?

Reply:
splitte opp denne i to scenario også?